src/user-guide/running-workflows/scheduler-log-files.rst

```diff
@@ -24,11 +24,50 @@ The information logged here includes:
 
    Workflow log files are primarily intended for human eyes. If you need
    to have an external system to monitor workflow events automatically, use:
-   
+
    * Event hooks (see :cylc:conf:`flow.cylc[scheduler][events]` and
      :cylc:conf:`flow.cylc[runtime][<namespace>][events]`).
    * The GraphQL interface (can be accessed via GraphiQL in the Cylc GUI).
    * The sqlite *workflow run database*
      (see :ref:`Workflow Run Databases`)
-     
+
    Rather than parse the log files.
+
+
+.. _scheduler Logs.Cylc message:
+
+Cylc Message
+^^^^^^^^^^^^
+
+.. warning::
+
+   Using Cylc Message to add information to the scheduler logs should
+   be used sparingly to avoid overwhealming users with output.
```

Comment:
A warning makes this look scary. I'd be inclined to let users decide what level of logging makes sense and scratch these two lines.